### DISCUSSION TITLE
"AR Quicklook" loads first AR model, but never the next "Zero KB" issue.

### DISCUSSION THREAD

**Discussion by phreak** [google/model-viewer#4846] [Q&A]
Hi all,

I created a viewer that allows switching between several 3D files in MV with buttons. Essentially its switching through models. 

When I start the first model in "Quicklook AR" it works fine. Only, every second model or the following are not starting in AR and instead showing a screen saying "Preview… Universal Scene Description Preview Package Zero KB" and Quicklook returns to the browser. The AR experience is not loaded. 

Juggling through the models to let another one start as the first model, doesn't help much either. The first AR experience is loaded, the others not.

This issue can easily be reproduced on iPhone's (here 14 Pro), although it works on Android and iPad Pro's just like it's meant to be:
https://cloud.arkid.app/brand/myjunior/myJunior-frames-iframe.html

iPhone device with the issues appearing:

- iPhone 14 Pro, iPhone 12, iPhone 7
- iOS: 17.6
- MV: 3.5.0

Anyone experienced this before?

**Comment by elalish** [comment]
That's a new one - @AdaRoseCannon any thoughts on what's broken here for quick look AR?

**Comment by AdaRoseCannon** [comment]
It's peculiar that it's different on iPad and iPhone definitely file a bug with feedback assistant.

**Comment by phreak** [comment]
@elalish and @AdaRoseCannon: Thank you for your reply. I filed a similar issue before here. https://feedbackassistant.apple.com/feedback/14610483

Sorry to mention this. I know we can't expect an answer. But by chance it would be lovely if you could look at the examples in the feedback assistant, (it's an A brand among them), and I would be delighted if you could highlight it to your colleagues. It's so hurtful that the 🍏 eco-system lacks so much in AR fidelity. 😓

**Reply by elalish** [reply]
Yeah, this is the big problem with Feedback Assistant - on WebKit or Github there's an actual conversation and you get notified when things are fixed. One-way communication is pretty inadequate for bug reporting.
